### ISSUE TITLE
[1.0.5 Patch][Flaky test]: Microsoft.DotNet.Watcher.Tools.FunctionalTests.AppWithDepsTests

### ISSUE THREAD

**Issue by pranavkm** [aspnet/DotNetTools#290]
Test was stuck for over 13 hours. Could we ensure there aren't any race conditions or checks we're missing that are causing it to be stuck? It would be nice to address this even once we get a patch build just so that it doesn't affect future patches.

**Comment by pranavkm** [comment]
Possible build log

```
The temporary test folder is C:\b\w\d6337ac2abf63a4b\.repositories\DotNetTools\testWorkDir\5adedf3314b24cc188b41e011b2fb747
Copying project C:\b\w\d6337ac2abf63a4b\.repositories\DotNetTools\test\Microsoft.DotNet.Watcher.Tools.FunctionalTests\bin\Release\netcoreapp1.1\TestProjects\GlobbingApp to C:\b\w\d6337ac2abf63a4b\.repositories\DotNetTools\testWorkDir\5adedf3314b24cc188b41e011b2fb747\work\GlobbingApp
Restoring msbuild project in GlobbingApp
Building GlobbingApp
4/28/2017 3:43:49 AM: process start: 'C:\b\w\d6337ac2abf63a4b\.build\dotnet\dotnet.exe exec --depsfile C:\b\w\d6337ac2abf63a4b\.repositories\DotNetTools\test\Microsoft.DotNet.Watcher.Tools.FunctionalTests\bin\Release\netcoreapp1.1\Microsoft.DotNet.Watcher.Tools.FunctionalTests.deps.json --runtimeconfig C:\b\w\d6337ac2abf63a4b\.repositories\DotNetTools\test\Microsoft.DotNet.Watcher.Tools.FunctionalTests\bin\Release\netcoreapp1.1\Microsoft.DotNet.Watcher.Tools.FunctionalTests.runtimeconfig.json --fx-version 1.0.5 C:\b\w\d6337ac2abf63a4b\.repositories\DotNetTools\test\Microsoft.DotNet.Watcher.Tools.FunctionalTests\bin\Release\netcoreapp1.1\dotnet-watch.dll --list'
4/28/20
```17 3:43:51 AM: post: 'C:\b\w\d6337ac2abf63a4b\.repositories\DotNetTools\testWorkDir\5adedf3314b24cc188b41e011b2fb747\work\GlobbingApp\Program.cs'
4/28/2017 3:43:51 AM: post: 'C:\b\w\d6337ac2abf63a4b\.repositories\DotNetTools\testWorkDir\5adedf3314b24cc188b41e011b2fb747\work\GlobbingApp\include\Foo.cs'
4/28/2017 3:43:51 AM: post: 'C:\b\w\d6337ac2abf63a4b\.repositories\DotNetTools\testWorkDir\5adedf3314b24cc188b41e011b2fb747\work\GlobbingApp\GlobbingApp.csproj'
4/28/2017 3:43:51 AM: recv: 'C:\b\w\d6337ac2abf63a4b\.repositories\DotNetTools\testWorkDir\5adedf3314b24cc188b41e011b2fb747\work\GlobbingApp\Program.cs'
4/28/2017 3:43:51 AM: recv: 'C:\b\w\d6337ac2abf63a4b\.repositories\DotNetTools\testWorkDir\5adedf3314b24cc188b41e011b2fb747\work\GlobbingApp\include\Foo.cs'
4/28/2017 3:43:51 AM: recv: 'C:\b\w\d6337ac2abf63a4b\.repositories\DotNetTools\testWorkDir\5adedf3314b24cc188b41e011b2fb747\work\GlobbingApp\GlobbingApp.csproj
```

**Comment by Eilon** [comment]
@muratg - can you assign to investigate? This was affecting the patch build.

**Comment by muratg** [comment]
@natemcmaster could you investigate?

**Comment by natemcmaster** [comment]
Fixed by #292. See CI build 10415
